main.js

```diff
@@ -0,0 +1,200 @@
+        person: document.getElementById("comment-name").value,
+        message: document.getElementById("comment-message").value,
+        ID: postID.toString(),
+        status: "live", // deleted comments will have a value of 'deleted'
+      };
+
+      allPosts[i].comments.push(commentObject);
+    };
+
+    const commentSubmitButton = document.createElement("button");
+    commentSubmitButton.id = "submit-comment";
+    commentSubmitButton.class = "btn btn-primary";
+    commentSubmitButton.innerHTML = "Submit Comment";
+    commentForm.appendChild(commentSubmitButton);
+
+    const renderComments = () => {
+      const commentUl = document.createElement("ul");
+
+      // for (let c = allPostDivs[i].length - 1; c >= 0; c--) {
+      //   commentSection.removeChild(allPostDivs[i]);
+      // }
+
+      for (let c = 0; c < allPosts[i].comments.length; c++) {
+        const commentBy = document.createElement(
+          "span",
+          allPosts[i].comments[c].person
+        );
+
+        const commentLi = document.createElement("li");
+
+        const x = document.createElement("button");
+        x.innerHTML = "X";
+        x.className = "btn btn-link";
+        x.style.fontWeight = "bold";
+
+        x.addEventListener("click", function () {
+          commentUl.removeChild(commentLi);
+        });
+
+        commentBy.style.fontWeight = "500";
+        commentBy.innerHTML = ` - Comment By: ${allPosts[i].comments[i].person}`;
+
+        const commentMessage = document.createElement("span");
+        commentMessage.innerHTML = allPosts[i].comments[c].message;
+
+        commentLi.appendChild(x);
+        commentLi.appendChild(commentMessage);
+        commentLi.appendChild(commentBy);
+        commentUl.appendChild(commentLi);
+
+        commentSection.appendChild(commentUl);
+      }
+    };
+    commentsButton.addEventListener("click", () => {
+      //toggle comments secotn on off while toggling the posts form off and on
+    });
+
+    commentSubmitButton.addEventListener("click", () => {
+      addComment();
+      renderComments();
+    });
+
+    commentSection.appendChild(commentForm);
+
+    renderComments();
+
+    newPost.appendChild(remove);
+    newPost.appendChild(commentsButton);
+    newPost.appendChild(postMessage);
+    newPost.appendChild(postOwner);
+    newPost.appendChild(commentSection);
+
+    // newPost.appendChild(hr);
+
+    remove.addEventListener("click", function () {
+      allPosts.splice([i], 1);
+      postsDiv.removeChild(newPost);
+    });
+
+    if (
+      document.getElementById("name").value &&
+      document.getElementById("message").value
+    ) {
+      postsDiv.appendChild(newPost);
+    } else {
+      alert("Please enter a message and your name.");
+    }
+  }
+};
+
+renderPosts();
+
+postButton.addEventListener("click", (event) => {
+  event.preventDefault();
+  addPost();
+});
+postButton.addEventListener("click", (event) => {
+  event.preventDefault();
+  renderPosts();
```

Comment:
You can combine these two event listeners since they're the same event ("click") on the same element (`postButton`).

Reply:
Got it!